plugins/inventory/efficientip.py

```diff
@@ -0,0 +1,360 @@
+# -*- Mode: Python; python-indent-offset: 4 -*-
+#
+# Time-stamp: <2020-04-18 16:19:07 alex>
+#
+
+"""
+ EfficientIP inventory for ansible
+ SOLIDserver Device Manager connection to gather devices filtered
+"""
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import ipaddress
+# import pprint
+import logging
+
+from ansible.plugins.inventory import BaseInventoryPlugin
+from ansible.plugins.inventory import Constructable, Cacheable
+from ansible.utils.display import Display
+from ansible.errors import AnsibleError
+
+try:
+    from SOLIDserverRest import adv as sdsadv
+    from SOLIDserverRest import SDSError
+except ImportError:
+    raise AnsibleError('The efficientip dynamic inventory'
+                       ' plugin requires SOLIDserverRest')
+
+logging.basicConfig(format='[%(filename)s:%(lineno)d]'
+                    ' %(levelname)s: %(message)s',
+                    level=logging.INFO)
+
+DOCUMENTATION = '''
+    name: efficientip
+    plugin_type: inventory
+    short_description: inventory based on EfficientIP IPAM
+    requirements:
+        - SOLIDserverRest > 2.0.1
+        - python 3
+    extends_documentation_fragment:
+        - inventory_cache
+    description:
+        - Get inventory hosts from EfficientIP IPAM and Device Manager
```

Comment:
```suggestion
        - Get inventory hosts from EfficientIP IPAM and Device Manager.
```
Descriptions should always be sentences.